var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -101,6 +109,56 @@ class Llvm(CMakePackage):
     base_url = 'http://llvm.org/releases/%%(version)s/%(pkg)s-%%(version)s.src.tar.xz'
     llvm_url = base_url % {'pkg': 'llvm'}
 
+    # Flang has a special version of clang named flang-driver.
```

Comment:
It's interesting that the projects have diverged but are similar enough that this package works for the most part on each. I'm not sure I know of any other cases of that in Spack.

Any chance you could consolidate these resources similar to how `compiler-rt` etc. are created below? I could see that being onerous since one of these resources identifies a branch vs. a commit, so the change isn't required.

Reply:
It is a good idea to group these resources declarations similar to what llvm does with its resources, and I will do that. It will be more convenient when we add more releases for flang-driver.